base/doc/ltnews41.tex

```diff
@@ -597,6 +597,32 @@ \subsection{Collecting environment bodies verbatim}
 \end{verbatim}
 \end{center}
 
+\subsection{Simple key=value argument processor}
+
+For very simple key=value setups with only a handful simple keys a new argument
+processor \cs{SplitKeyval} is added that works similar to \cs{SplitArgument} but
+using \meta{key}\verb|=|\meta{value} pairs instead of a splitting symbol. The
+processor takes the list of known keys as its mandatory argument, the fallback
+values will be the special \verb|-NoValue-| marker or the value given to it when
+it is declared. The values will be forwarded in the same order the keys were
+declared. Thus
+\begin{verbatim}
+  \NewDocumentCommand\MyKeyvalCmd
+    {>{\SplitKeyval{keyA=A,keyB,key=C}}m}
+    {\FunctionOfThree#1}
+\end{verbatim}
+defines a macro that knows three keys. If during usage any key doesn't get a
+value an empty value will be forwarded. We can then use the macro and get the
```

Comment:
```suggestion
value an empty value, will be forwarded. We can then use the macro and get the
```

Reply:
I think the comma should be after the first "value" here, not after the second?

Reply:
yes, sorry